packages/atomic/cypress/e2e/result-list/result-components/quickview.cypress.ts

```diff
@@ -142,6 +142,25 @@ describe('Quickview Component', () => {
     });
   });
 
+  describe('when used on pdf file inside a result list with exact phrase match', () => {
+    beforeEach(() => {
+      new TestFixture()
+        .with(addFacet({field: 'author'}))
+        .with(addSearchBox())
+        .withHash('q="the%20bank"&f-author=Jules%20Verne')
+        .with(addQuickviewInResultList())
+        .init();
+    });
+
+    it('should display only highlight button for the phrase with multiple occurrences', () => {
```

Comment:
does this test that the exact phrase is highlighted?

Reply:
It performs a query for the exact phrase "the bank", with author=Jules Verne (line 150) (a pdf book for around the world in 90 days).

Then looks at the content of the highlight button, which says there's 23 occurrences for the exact phrase "the bank"

Reply:
Ok got it so relies on the summary text on number of occurrences rather than verifying that the text is present and highlighted?

Reply:
Yeah, otherwise we'd have to replicate the same logic as the component directly in the test: As explained in the summary, the keyword highlighting is not exactly matched, it relies on stemming information from the index. 

And so we'd have to add complex-ish logic in the test file that's very similar to what's in `quickview-word-highlight` if we wanted to verify how keywords are highlighted inside the iframe.

VS verifying that the summary is correct seems much simpler.